package.json

```diff
@@ -25,14 +25,15 @@
     "@fortawesome/free-solid-svg-icons": "^6.5.2",
     "@fortawesome/react-fontawesome": "^0.2.2",
     "@headlessui/react": "^2.0.4",
-    "@heroicons/react": "^2.1.3",
+    "@heroicons/react": "^2.1.4",
     "@ngneat/falso": "^7.2.0",
     "@prisma/client": "^5.14.0",
     "@tailwindcss/forms": "^0.5.7",
     "next": "14.2.3",
     "next-auth": "^4.24.7",
     "react": "^18",
     "react-dom": "^18",
+    "react-icons": "^5.2.1",
```

Comment:
We don't need a whole new library for 1 icon

e now have font awesome in the project, that has the github icon, please update your fork